### ISSUE TITLE
Can't install pact_verifier_cli crate

### ISSUE THREAD

**Issue by vlastimil-dolejs** [pact-foundation/pact-reference#123]
I have used a fresh DigitalOcean machine with Ubuntu 20.04.2 (the same happens on my Windows machine in Docker build and WSL)

```
apt-get update 
apt-get install -y build-essential curl
curl https://sh.rustup.rs -sSf | bash -s -- -y
PATH="/root/.cargo/bin:${PATH}"
cargo install --version 0.8.5 pact_verifier_cli
```

Result is:
```
 Compiling pact_verifier_cli v0.8.5
error[E0061]: this function takes 1 argument but 0 arguments were supplied
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/pact_verifier_cli-0.8.5/src/main.rs:242:9
    |
242 |   match handle_cli().await {
    |         ^^^^^^^^^^-- supplied 0 arguments
    |         |
    |         expected 1 argument
    |
note: function defined here
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/pact_verifier_ffi-0.0.6/src/verifier.rs:133:14
    |
133 | pub async fn handle_cli(version: &str) -> Result<(), i32> {
    |              ^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0061`.
error: failed to compile `pact_verifier_cli v0.8.5`, intermediate artifacts can be found at `/tmp/cargo-install2dmRC8`

Caused by:
  could not compile `pact_verifier_cli`
```

**Comment by uglyog** [comment]
I've released 0.8.6 which is fixed

**Comment by vlastimil-dolejs** [comment]
Thank you. I can confirm that the issue is resolved.
